testing/src/main/resources/certs/README

```diff
@@ -59,11 +59,25 @@ $ rm server1.key.rsa
 $ openssl req -new -key server1.key -out server1.csr -config server1-openssl.cnf
 
 When prompted for certificate information, everything is default except the
-common name which is set to *.test.google.com.
+common name, which is set to *.test.google.com.
 
 $ openssl ca -in server1.csr -out server1.pem -keyfile ca.key -cert ca.pem -verbose -config server1-openssl.cnf -days 3650 -extensions v3_req -updatedb
 $ openssl x509 -in server1.pem -out server1.pem -outform PEM
 
+localhost_server is issued by CA with the common name "localhost":
```

Comment:
Right, the purpose of adding the cert was simply to have the "localhost" CN. I've made the change you suggest. Curiously, this change caused the `serverRejectsUntrustedClientCert` test to start failing, just like in issue #1325.